react/week3/react-Week-Three-HW/todo-app/src/Components/Todo.js

```diff
@@ -0,0 +1,196 @@
+import React,{Component} from "react";
+import PropTypes from "prop-types";
+
+export const RenderList = (props) => {
+	const editstate = props.todos.editstate;
+
+	const details = editstate ? (
+		<input
+			type="text"
+			placeholder="update todo"
+			onChange={(event) => {
+				const updatedEvent = event.target.value;
+
+				return props.updatedTodo(updatedEvent);
+			}}
+		/>
+	) : (
+		` ${props.todos.description}  ||  ${props.todos.deadline}`
+	);
+
+	const todoDone = props.todos.completed;
```

Comment:
Stick with the naming. If you call them completed then keep calling the completed. Sometimes calling it done and sometimes completed will give problems.